src/pages/index.tsx

```diff
@@ -0,0 +1,24 @@
+import Head from "next/head";
+import { Inter } from "next/font/google";
+import styles from "@/styles/Home.module.css";
+import dynamic from "next/dynamic";
+
+const inter = Inter({ subsets: ["latin"] });
+
+const AppWithoutSSR = dynamic(() => import("@/App"), { ssr: false });
```

Comment:
**suggestion**: Consider adding error handling for the dynamic import to manage potential loading issues.